frontend/src/component/admin/instance-privacy/InstancePrivacy.tsx

```diff
@@ -0,0 +1,147 @@
+        Projects: 'The number of projects in your instance',
+        'Context Fields': 'The number of custom context fields in use',
+        Groups: 'The number of groups present in your instance',
+        Roles: 'The number of custom roles defined in your instance',
+        Environments: 'The number of environments in your instance',
+        Segments: 'The number of segments defined in your instance',
+        Strategies: 'The number of strategies defined in your instance',
+        'Feature Exports': 'The number of feature exports performed',
+        'Feature Imports': 'The number of feature imports performed',
+        'Custom Strategies':
+            'The number of custom strategies defined in your instance',
+        'Custom Strategies In Use':
+            'The number of custom strategies that are in use by feature toggles',
+    },
+};
+
+const versionCollectionActivenessManagementTexts: IFeatureActivenessManagementInfo =
+    {
+        enabled: {
+            environmentVariables: 'CHECK_VERSION=false',
+            changeInfoText:
+                'Version info collection can be disabled by setting the environment variable `CHECK_VERSION` to `false` and restarting Unleash.',
+        },
+        disabled: {
+            environmentVariables: 'CHECK_VERSION=true',
+            changeInfoText:
+                'Version info collection can be enabled by setting the environment variable to true and restarting Unleash.',
+        },
+    };
+
+const featureCollectionActivenessManagementTexts: IFeatureActivenessManagementInfo =
+    {
+        enabled: {
+            environmentVariables: 'SEND_TELEMETRY=false',
+            changeInfoText:
+                'Feature usage collection can be disabled by setting the environment variable to false and restarting Unleash.',
+        },
+        disabled: {
+            environmentVariables: 'SEND_TELEMETRY=true',
+            changeInfoText:
+                'To enable feature usage collection set the environment variable to true and restart Unleash.',
+        },
+    };
+
+export const InstancePrivacy = () => {
+    const { settings } = useTelemetry();
+    const { uiConfig, loading } = useUiConfig();
+
+    if (loading) {
+        return null;
+    }
+
+    const versionActivenessInfo = settings?.versionInfoCollectionEnabled
+        ? versionCollectionActivenessManagementTexts.enabled
+        : versionCollectionActivenessManagementTexts.disabled;
+
+    const featureActivenessInfo = settings?.featureInfoCollectionEnabled
+        ? featureCollectionActivenessManagementTexts.enabled
+        : featureCollectionActivenessManagementTexts.disabled;
+
+    let dependsOnFeatureCollection: undefined | string = undefined;
+    if (!settings?.versionInfoCollectionEnabled)
+        dependsOnFeatureCollection = settings?.featureInfoCollectionEnabled
+            ? 'Note: Feature Usage Collection is enabled, but for it to be active you must also enable Version Info Collection'
```

Comment:
Here "Feature Usage Collection" uses title case, however in https://github.com/Unleash/unleash/pull/4035/files#diff-5d0985ea6569ae4ed217ebe878d451ea76a87e5e7e14a9dabde9230a2de24aa1R77 it doesn't. Should we try to make it consistent?